pallets/loans-ref/src/types.rs

```diff
@@ -767,4 +767,10 @@ mod test_utils {
 			self
 		}
 	}
+
+	impl<T: Config> ActiveLoan<T> {
+		pub fn set_maturity(&mut self, duration: Duration) {
+			self.info.schedule.maturity = Maturity::Fixed(duration.as_secs());
+		}
+	}
```

Comment:
Should this maybe also be guarded by benchmark feature flag?

Reply:
Yes, it already is